src/app/test_executive/zkapps.ml

```diff
@@ -496,27 +496,27 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
         let with_dummy_signatures =
           mk_forest
             [ mk_node
-                (mk_account_update_body Signature Call token_owner
+                (mk_account_update_body Signature No token_owner
                    Token_id.default
                    (-account_creation_fee_int) )
                 [ mk_node
-                    (mk_account_update_body Signature Call token_accounts.(1)
+                    (mk_account_update_body Signature No token_accounts.(1)
```

Comment:
`Parents_own_token` everywhere in this txn where `custom_token_id` or `custom_token_id2` is used